docs/tutorials/configure/scuttlebot.md

```diff
@@ -36,42 +36,11 @@ cost: 540
 
 ## Configure the board
 
-Add your first component, the [board](/components/board/):
+Add your first component, the [board](/components/board/).
```

Comment:
Discussion question: technically, I think you can now set up a scuttlebot without having a board component! Should we include this step at all? Part of me says "no, the minimum working example does not need a board component." Part of me says "yes, we should have components for every piece of hardware, and we should show people how to use the board if they want to use it for something else (e.g., they want to put an ultrasonic sensor on their scuttlebot)." and then the first part of me says "so don't introduce the board component until it's needed in a different tutorial. It's unused here." What do you think?

Reply:
I think it makes sense to remove. removing!